src/gsy_e/gsy_e_core/cli.py

```diff
@@ -132,10 +133,13 @@ def run(setup_module_name, settings_file, duration, slot_length, tick_length,
                 external_connection_enabled=enable_external_connection,
                 enable_degrees_of_freedom=enable_dof)
 
+        gsy_e.constants.SEND_EVENTS_RESPONSES_TO_SDK_VIA_RQ = False
```

Comment:
Since the default is already `False`, do you think we can omit this line?

Reply:
Probably, yes I wanted to be explicit, but no need, will remove it.